test/lib/ufe/testDeleteCmd.py

```diff
@@ -267,7 +267,7 @@ def testDeleteHierarchy(self):
         # delete two USD prims and Maya's shape
         ufeObs.reset()
         cmds.delete('|Tree_usd|Tree_usdShape,/TreeBase')
-        self.assertEqual(ufeObs.nbDeleteNotif() , 1)
+        self.assertEqual(ufeObs.nbDeleteNotif() , 4)
```

Comment:
This is quite a shame.  We should only notify once, at the root of the sub-graph, when deleting a single prim.  Otherwise, for a large scene, this will not be scalable.  An explicit deletion list, either through a command line argument, or a selection list, should generate one notification per prim, but not here.

Perhaps an SdfChangeBlock will help?

Reply:
Yep, a change block would help and make it match what redo does...but now it doesn't matter since recursive removal is gone.